stories/header.stories.js

```diff
@@ -37,13 +37,17 @@ storiesOf( 'Global', module )
 		return storyFn( registry );
 	} )
 	.add( 'Plugin Header', ( registry ) => {
+		global.featureFlags = { storeErrorNotifications: { enabled: true } };
+
 		return (
 			<WithTestRegistry registry={ registry }>
 				<Header />
 			</WithTestRegistry>
 		);
 	} )
 	.add( 'Plugin Header with Date Selector', ( registry ) => {
+		global.featureFlags = { storeErrorNotifications: { enabled: true } };
```

Comment:
We need a better way to work with feature flags for Storybook, but for now, let's only set the one flag rather than overwrite the entire object. (Same above)

This is really easy using `lodash.set` – e.g. https://github.com/google/site-kit-wp/pull/2427/files#diff-b784d442e62f68075556b9485703ee43928f2ec379430fbfb7983001aa9cbae1R40